doc-content/enterprise-only/springboot/spring-boot-jms-audit-proc.adoc

```diff
@@ -0,0 +1,126 @@
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+----
+
+.. Add the following lines to the `application.properties` file of the service that will consume the message broker data:
++
+[source]
+----
+kieserver.audit-replication.consumer=true
+kieserver.audit-replication.queue=audit-queue
+----
+.. Add the properties required for your message broker client to the `application.properties` file of the service that will consume the message broker data. The following example shows how to configure {KIE_SERVER} for AMPQ where  `<JMS_HOST_PORT>` is the port that your message broker listens on and `<USERNAME>` and `<PASSWORD>` are the login credentials for the message broker:
++
+[source]
+----
+amqphub.amqp10jms.remote-url=amqp://<JMS_HOST_PORT>
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+----
+. To configure {KIE_SERVER} audit replication to use topics, complete the following tasks:
++
+.. Add the following lines to your Spring Boot application's `application.properties` file:
++
+[source]
+----
+kieserver.audit-replication.producer=true
+kieserver.audit-replication.topic=audit-topic
+----
+.. Add the properties required for your message broker client to the `application.properties` file of the service that will consume the message broker data. The following example shows how to configure {KIE_SERVER} for AMPQ where `<JMS_HOST_PORT>` is the port that your message broker listens on and `<USERNAME>` and `<PASSWORD` are the login credentials for the message broker:
++
+[source]
+----
+spring.jms.pub-sub-domain=true
+amqphub.amqp10jms.remote-url=amqp://<JMS_HOST_PORT>
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+----
+
+.. Add the following lines to the `application.properties` file of the service that will consume the message broker data:
++
+[source]
+----
+kieserver.audit-replication.consumer=true
+kieserver.audit-replication.topic=audit-topic::jbpm
+kieserver.audit-replication.topic.subscriber=jbpm
+spring.jms.pub-sub-domain=true
+----
+
+
+.. Add the properties required for your message broker client to the `application.properties` file of the service that will consume the message broker data. The following example shows how to configure {KIE_SERVER} for AMPQ where `<JMS_HOST_PORT>` is the port that your message broker listens on and `<USERNAME>` and `<PASSWORD>` are the login credentials for the message broker:
++
+[source]
+----
+amqphub.amqp10jms.remote-url=amqp://<JMS_HOST_PORT>
+amqphub.amqp10jms.username=<USERNAME>
+amqphub.amqp10jms.password=<PASSWORD>
+amqphub.amqp10jms.pool.enabled=true
+amqphub.amqp10jms.clientId=jbpm
+----
+
+. To configure the {KIE_SERVER} that contains the replicated data to be read only, set the `org.kie.server.rest.mode.readonly` property in the `application.properties` file to `true`:
```

Comment:
This step is optional. Not really require to setup when using JMS audit replication. I would mention something like:
```suggestion
. Optionally, to configure the {KIE_SERVER} that contains the replicated data to be read only, set the `org.kie.server.rest.mode.readonly` property in the `application.properties` file to `true`:
```